scripts/in_container/run_tmux.sh

```diff
@@ -21,6 +21,7 @@ if [[ ${START_AIRFLOW:="false"} == "true" ]]; then
 
     export AIRFLOW__CORE__LOAD_DEFAULT_CONNECTIONS=${LOAD_DEFAULT_CONNECTIONS}
     export AIRFLOW__CORE__LOAD_EXAMPLES=${LOAD_EXAMPLES}
+    export AIRFLOW__CORE__EXECUTOR=${AIRFLOW__CORE__EXECUTOR:-LocalExecutor}
```

Comment:
I think it cannot be set here for SQLITE case

Reply:
oh yes, ofcourse, let me add some conditionals

Reply:
I've added it here: https://github.com/apache/airflow/pull/11792

Reply:
Probably better to add it in the docker-compose config

Reply:
But still there is the lock in sqllite :(

Reply:
I don't want to do in docker-compose file as it will use that executor for running tests in CI too

Reply:
But shoudl not it be the case when you enter breeze shell?

Reply:
>But still there is the lock in sqllite :(

Yeah we will need to find and fix for sqlite too, but this was mostly for users using `breeze start-airflow` for doing rc testing etc

Reply:
I think Local executor should be default if not set - this might be a better approach

Reply:
>But shoudl not it be the case when you enter breeze shell?

I am mainly thinking no for that, just so that users can replicate the stuff in CI locally when they enter just `./breeze`.

but when do they do `breeze start-airflow` it is for running some DAGs instead of just running unit-tests

Reply:
And we should fail hard if executor == SEQUENTIAL + db is anything else than sqlite (as long as we fix sqlite)

Reply:
So far what many people were doing when entering breeze they run tmux and then "airflow webserver" and "airlfow scheeduler" and they run dag testing. It would be surprising tot see a hanging run there without explanation.

"start-airflow" is rather new addition (2 weeks).

Reply:
The CI tests run so far without having to set the executor - this is what I am really about it, so changing to a non-working executor by default, when it was working fine before is not a good idea.

Reply:
we definitely need to fix sqlite as that is essential so for those users who run `tmux` from within `./breeze` it should still work. But we do recommend people to use `./breeze start-airflow` in docs too because it supports using an ENV file too and I feel it is very powerful because of that and coz it also support and init file

Reply:
>The CI tests run so far without having to set the executor - this is what I am really about it, so changing to a non-working executor by default, when it was working fine before is not a good idea.


Yeah, don't get me wrong, I am going to debug the sqlite issue too, this is just an optimization for people who just wants to test

Reply:
Also - the SequentialExecutor should not be default I think :( seems it is right now (but was not)

Reply:
@potiuk you are confusing few things here. the bug with `SequentialExecutor` needs to be fixed as I said in my previous comments. This PR has nothing to do with fixing that bug, this is just an optimization like explained in PR description.

The bug has to be fixed separatly